lib/sonice/public/script.js

```diff
@@ -116,6 +116,19 @@ $(function() {
     })
   }
 
+  // Toggle the playpause button on the remote button widget
+  var togglePlayPause = function(toggle_playpause_button){
```

Comment:
Identifier 'toggle_playpause_button' is not in camel case.<br>Too many errors. (49% scanned).